tests/lib/SimpleSAML/Utils/AttributesTest.php

```diff
@@ -15,33 +16,33 @@ class AttributesTest extends TestCase
     /**
      * Test the getExpectedAttribute() method with invalid attributes array.
      * @return void
+     * @psalm-suppress InvalidArgument
+     * @deprecated Can be removed as soon as the codebase is fully typehinted
      */
     public function testGetExpectedAttributeInvalidAttributesArray()
     {
         // check with empty array as input
         $attributes = 'string';
         $expected = 'string';
-        $this->expectException(
-            \InvalidArgumentException::class,
-            'The attributes array is not an array, it is: '.print_r($attributes, true).'.'
-        );
```

Comment:
Not sure in what PHPunit-version this was allowed, but today it's split in two calls (actually for years already)